kafka-native/src/main/java/org/ballerinalang/messaging/kafka/impl/KafkaListenerImpl.java

```diff
@@ -20,12 +20,11 @@
 
 import org.apache.kafka.clients.consumer.ConsumerRecords;
 import org.apache.kafka.clients.consumer.KafkaConsumer;
-import org.ballerinalang.jvm.api.BExecutor;
+import org.ballerinalang.jvm.api.BRuntime;
 import org.ballerinalang.jvm.api.connector.CallableUnitCallback;
 import org.ballerinalang.jvm.api.values.BObject;
 import org.ballerinalang.jvm.observability.ObservabilityConstants;
 import org.ballerinalang.jvm.observability.ObserveUtils;
-import org.ballerinalang.jvm.scheduling.Scheduler;
 import org.ballerinalang.jvm.scheduling.Strand;
```

Comment:
Unused import? (Shall we add `checkstyle` and `spotbugs` to the Kafka module? Not with this PR, but later?)

Reply:
Yes. https://github.com/ballerina-platform/module-ballerinax-kafka/issues/49